tests/cases/compiler/writerOptionalIndentation.ts

```diff
@@ -0,0 +1,93 @@
+// @module: commonjs
+// @skipLibCheck: true
+// @includebuiltfile: typescriptServices.d.ts
+// @noImplicitAny:true
+// @strictNullChecks:true
+
+// @filename: node_modules/typescript/index.d.ts
+declare module "typescript" {
+    export = ts;
```

Comment:
I would look at the API tests instead; this isn't how these tests are written anymore as `typescriptServices.d.ts` is going away in a future release.

Reply:
tests rewritten in the API test style